eonx-com/easy-security/2.3/src/Security/Interfaces/RolesInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Security\Interfaces;
+
+interface RolesInterface
```

Comment:
@Nyholm Thank you for looking at the PR. The interfaces for roles and permissions are a "convention" we started to use to centralise the definition of roles/permissions in the application and also to allow these roles/permissions to be reused in the code in a consistent way.

The application doesn't have to use it but we recommend doing it this way, that's why we include these empty interfaces as part of the recipe.

Reply:
Sorry for being late to the party, but if you don't force to use this interface, which is fine, it should belong to the code repository and not the recipe

Reply:
@OskarStark it's only been a few years it's ok 😄 Thank you for taking the time to look at it. I actually completely forgot about this PR so your comment reminded me of it, that's good.

We can actually close it, because since then we have migrated all our projects to using PHP for the application config, so these recipes are now irrelevant.

But once again, thanks for looking into it and bringing that back to my attention!